src/main/java/uk/gov/ons/ctp/response/casesvc/domain/repository/CaseGroupRepository.java

```diff
@@ -147,4 +137,14 @@ List<CaseAction> findByCollectionExerciseIdAndActiveEnrolment(
               + "WHERE cg.caseGroupPK=c.caseGroupFK AND c.state='ACTIONABLE' "
               + "AND c.id IN :caseIds ")
   List<CaseAction> findByCaseIdIn(@Param("caseIds") List<UUID> caseIds);
+
+  /**
+   * find cases for action
+   *
+   * @param collectionExerciseID UUID of the collection exercise to delete casegroup entities
+   */
```

Comment:
using the modifying and query annotations returns an int of deleted rows, the inbuilt delete returns void.